### DISCUSSION TITLE
Ability to sync favorites in Self Hosted Free

### DISCUSSION THREAD

**Discussion by gacpac** [rustdesk/rustdesk#7533] [General]
My use case is very simple. I support my family and friends and recently migrated from teamviewer free (kept getting the annoying business flag)

Any way I can somehow sync or backup favorites in the agent? That way i can connect from a different machine or phone at times.  Manually the peers file can be saved and moved around, maybe a way to pull it from somewhere else without needing a pro license for home usage.

**Comment by GuillaumeHullin** [comment]
Well, that's the purpose of the Pro version with the support of address book.
Here is my workaround: Bitwarden (or any password manager).
Your password manager is sync between your device and securely store your friends and family computer access. 

Also maybe take a look at https://github.com/rustdesk/rustdesk/issues/616 for the use of URI.

**Reply by gacpac** [reply]
you know i think i got your point. how i didn't think of it, i'll store the IDs in my bitwarden and call it a day

**Comment by cvpsmith** [comment]
Alternatively, you could use an rsync script on Linux (or something like FreeFileSync on Windows) to sync the directories between systems.

**Reply by GuillaumeHullin** [reply]
I'm not fully sure what @gacpac means by "peers file", is there a way to synchronise conf accross systems?

**Comment by gacpac** [comment]
I mean the peers in the settings. What I've been doing is I have the IDs
saved somewhere else

**Comment by GuillaumeHullin** [comment]
Ok understood. I thought you managed to have some conf files that you could install across different systems and have the ID in sync ;)

**Comment by gacpac** [comment]
Thay would be lovely but I guess that is what the paid version is for

**Reply by GuillaumeHullin** [reply]
Indeed ;)

**Comment by RickkeeC** [comment]
Hello, I have 300 PC's, interested in self-hosted. I just need a phone book.  It looks like the fee is $49.90 / month + .10 for each additional PC, just so I can have a phone book.  Is this correct? $598.80 seems high, it is the same I am paying now for current vendor with unlimited computers and 10 users.

**Reply by cvpsmith** [reply]
> I just need a phone book

So, it sounds like all that you need is a database of some sort.  One where you can store URL links to these RustDesk entries that look like this: "rustdesk://connection/new/123456789" ("123456789" being an example of the RustDesk ID).

You can use ANY database-type program for that.  I'd suggest a secure one such as [KeePassXC](https://keepassxc.org/), or one of the other password managers suggested by @GuillaumeHullin above.

**Comment by RickkeeC** [comment]
Thank you.  Too cumbersome for more than a few dozen records. I have 300 and growing. What I need is righteous price for phone book feature for self hosted.  Anydesk, Splashtop other established competitors are the same. Iperius and Anyview are less expensive.  Rustdesk not trusted by SonicWall and Zyxel firewalls reputation filters, tagged as malicious app have to create rule but cannot expect clients I do not manage  to do that. No support by email or phone for sales or support at rustydesk but $50 a month for a phone book is the deal breaker. Nice product though, I enjoyed working with it but will stick with investment in the more mature products for now.

**Reply by GuillaumeHullin** [reply]
Unlike Anydesk and others, RustDesk is open-source and so allows you to audit the code, so you know exactly what you’re running, rather than just trusting a company’s word.

Regarding firewalls, unless I'm mistaken you are talking about UPnP? Which I would want disabled in any professional network... 

As for the phone book feature, I do prefer keeping sensitive access credentials in a password manager, not within the server for remote access... yes it's more work but I do think there is less chance to have a security flaw in Bitwarden running local than in RustDesk on Internet. 

And to my point look what happened this February... https://anydesk.com/en/faq-incident
I do not mean to hit hard on Anydesk... I do think they handled it correctly and it's not Crowdstrike level. However that reinforce my decision of keeping access management outside the server.

**Reply by cvpsmith** [reply]
> yes it's more work but I do think there is less chance to have a security flaw in Bitwarden running local than in RustDesk on Internet

That is a VERY good point!

**Comment by cvpsmith** [comment]
>  No support by email or phone for sales or support at rustydesk but $50 a month for a phone book is the deal breaker.

I'm not sure where you're getting that $598 figure from that you previously mentioned, but Rustdesk Pro is nowhere near that for 300 systems/10 users.  It's $49.90 per month for that subscription, and that's significantly cheaper than you will find for any other software for this type.  And trust me, I looked.  For example, Splashtop would cost you $82.50 for the same license type, and they have VERY poor Linux support compared to Rustdesk.

It sounds like you have some very specific support needs with regards to client firewalls, as well as a need for software email/phone support.  These things cost money.  $50/month to support 300 PCs in a commercial setting is an absolute bargain in my opinion.

**Comment by RickkeeC** [comment]
Hello,

“I'm not sure where you're getting that $598 figure from that you previously mentioned, but Rustdesk Pro is nowhere near that for 300 systems/10 users”
$49.90 x 12 months = $598.90

“$50/month to support 300 PCs in a commercial setting is an absolute bargain in my opinion.”
Its not bad, it is on-par with some others.

RustDesk is a great product, all have advantages and disadvantages.
I am just saying that to me, it is not worth $598.90 / year just to have a phone book.
SUGGESTION:  Would be willing to pay up to $199 / year for a phone book.
SUGGESTION:  Get RustDesk off the bad IP reputation list.  The NYC relay was the one that got blocked. I understand with my own server that would not be a problem.

RustDesk - $49.90 X 12 = 598.80 per year. + USD $0.10 per additional device.
Splashtop SOS, $399 / year / PER TECH
AnyViewer - $199 / year 30 Techs / 100 Login Devices (one-click) / 300 access by code, can also be set to always on, but no one click.  The on-demand entries can be added to the address book 😊 and placed into groups with ACL technician access.   Waiting on cost for additional one-click devices.  Current Promo 3 years subscription for $358.  I got mine for $299 with a in-trial banner ad.  Have 15 days to decide if it’s a keeper.  Today, this appears to be the absolute bargin. As long as you do  not need Linux.
Iperius Remote - $440 USD / year 10 Concurrent connections, unlimited devices, and techs

None of the above have Linux, I don’t need Linux support, use SSH for that.

RC

**Reply by cvpsmith** [reply]
> $399 / year / PER TECH

This, combined with poor Linux support is what made me move away from Splashtop.   I meant CLIENT Linux support here, so the Splashtop Business application.  All of my systems (desktop/laptop) are Linux-based, as are my servers.  The only Windows systems that I connect to/support are user clients.

I hadn't heard of AnyViewer, but it does sound like the best deal if it meets your needs.  As for Iperius, I'm not sure I like the idea of a license based on concurrent connections.  What are they, Netflix?  ;)
